tests/test_kraken3.py

```diff
@@ -142,3 +156,4 @@ def test_krakenz3_not_totally_broken(mock_krakenz3):
     _ = mock_krakenz3.get_status()
     mock_krakenz3.set_speed_profile(channel="fan", profile=iter([(20, 20), (30, 50), (40, 100)]))
     mock_krakenz3.set_fixed_speed(channel="pump", duty=50)
+    mock_krakenz3.set_screen("lcd", "liquid", None)
```

Comment:
Now I think we're at the point we need at least one test case for each of the screen features (including asserting the expected messages exchanged with the cooler).